.pipeline/config.yml

```diff
@@ -0,0 +1,12 @@
+steps:
```

Comment:
Just occurred to me: I assume you have tested this, and no "general" section is necessary?

Reply:
we have no general section here too https://github.com/SAP/jenkins-library/blob/master/.pipeline/config.yml

I think for the go binary this is ok. note this is not running in jenkins